spec/System/HapistranoSpec.hs

```diff
@@ -42,7 +43,11 @@ spec = do
            expectedOutput `Hspec.shouldSatisfy` (`isPrefixOf` actualOutput)
   describe "readScript" $
     it "performs all the necessary normalizations correctly" $ do
+#if MIN_VERSION_path_io(1,6,0)
+      let spath = $(mkRelFile "script/clean-build.sh")
+#else
       spath <- makeAbsolute $(mkRelFile "script/clean-build.sh")
+#endif
```

Comment:
I'm not really confident in this change @juanpaucar but I think it's fine since it's for testing.

The reason behind this is due to the change of types for `AbsPath` in [1.5.0](https://hackage.haskell.org/package/path-io-1.5.0/docs/Path-IO.html#t:AbsPath) from a `kind "*"` to a `Type` in [1.6.0](https://hackage.haskell.org/package/path-io-1.6.0/docs/Path-IO.html#t:AbsPath). If you have a better idea to solve this, I'm more than glad to hear it :smiley: